dnanexus/stjude_seaseq/dxapp.json

```diff
@@ -36,6 +36,13 @@
       "optional": false,
       "help": "Select from built-in list of genomes. Otherwise, select Custom Genome and provide 'Genome Reference' and 'Gene Annotation'."
     },
+    {
```

Comment:
The `run_motifs` option was intentionally not included in DNAnexus app. Is there a reason for including it here? 
Might be better to include the peak calling & downstream analysis (i.e. paired_sample_analysis) as optional.

Reply:
It was optional in `seaseq-control.wdl`, but the parameter wasn't exposed on DNAnexus. Can you clarify why it shouldn't be exposed on DNAnexus?

Reply:
SEAseq is built to perform Motif Analysis (i.e motif enrichment and discovery). However, our team (Abraham's lab) doesn't require this analysis, which is why the option is only provided in the WDL script.

Reply:
I can remove it if necessary, but given that the option exists, it seems to make sense to expose that to users. Others might have the same needs as the Abraham lab with respect to their analysis. With the default set to true, it doesn't change the behavior of the workflow, just gives users an option.